apps/laboratory/tests/shared/constants/index.ts

```diff
@@ -1,7 +1,9 @@
 import type { SessionParams } from '../types'
 
 // Allow localhost
-export const LOCAL_LABS_URL = 'http://localhost:3000/library/wagmi/'
+export const BASE_URL = 'http://localhost:3000/library/'
+// Allow localhost
+export const LOCAL_LABS_URL = `${BASE_URL}wagmi/`
```

Comment:
why set the base url to wagmi?

Reply:
`BASE_URL` now is not specific. `LOCAL_LABS_URL` is the entrypoint for the test and needs to be a website that doesn't HTTP404 so I chose this one. Could also use `http://localhost:3000/` I presume

Reply:
Fixed